featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -442,7 +442,8 @@ def update_progress_callback(done):
                             _feature_matrix[col] = pass_through[col]
                 elif isinstance(_feature_matrix, dd.core.DataFrame) and (len(pass_columns) > 0):
                     _feature_matrix['time'] = time_last
-                    _feature_matrix['time'] = dd.to_datetime(_feature_matrix['time'])
+                    if isinstance(time_last, pd.Timestamp):
```

Comment:
would it be sufficient to check the `time_type` of the entityset here, given that in `calculate_feature_matrix` we validate the the cutoff time time column's dtype is compatible with the time type of the entityset?

Reply:
After additional review, it seems this casting operation is not necessary any longer. I removed this entirely and also refactored a bit to remove a duplicated code block for creating `pass_through`.